tests/layer_tests/pytorch_tests/test_stft.py

```diff
@@ -138,10 +138,6 @@ def test_stft_not_supported_attrs(self, n_fft, hop_length, win_length, center, p
             pytest.xfail(
                 reason="torch stft uses list() for `center` subgrpah before aten::stft, that leads to error: No conversion rule found for operations: aten::list")
 
-        if normalized is True:
```

Comment:
Please move test case with normalized from unsupported section (line 129) + consider adding couple more test cases for normalized scenario